x-pack/plugins/canvas/public/components/workpad_filters/filter.scss

```diff
@@ -0,0 +1,13 @@
+.workpadFilter {
+  &.euiDescriptionList.euiDescriptionList--column > *:first-child,
+  &.euiDescriptionList.euiDescriptionList--column > :nth-child(2),
+  &.euiDescriptionList.euiDescriptionList--responsiveColumn > *:first-child,
+  &.euiDescriptionList.euiDescriptionList--responsiveColumn > :nth-child(2) {
+    margin-top: 0;
+  }
+
+  &.euiDescriptionList.euiDescriptionList--column > *,
+  &.euiDescriptionList.euiDescriptionList--responsiveColumn > * {
+    margin-top: 10px;
+  }
+}
```

Comment:
This looks curious to me. Can you explain the problem this was solving? I'm guessing there may be a simpler solution out there.

Reply:
Yea, let me explain) 

By default, `euiDescriptionList` class with columns has such a style.
![Screenshot 2021-11-09 at 12 43 20](https://user-images.githubusercontent.com/22456368/140909975-f5b9e3c8-bf8f-4bd1-9a40-8419ebe14355.png)

I wished to change `margin-top` for columns to `10px`. But that means it will rewrite the default `0px` `margin-top` of the first two children of the `euiDescriptionList` class (dd and dt). So I had to write that rule too.

Without styles:
![Screenshot 2021-11-09 at 12 48 00](https://user-images.githubusercontent.com/22456368/140910608-e2281b18-68e5-4ab6-869e-50672361eae1.png)

With styles:
![Screenshot 2021-11-09 at 12 48 51](https://user-images.githubusercontent.com/22456368/140910727-51b161ed-b27a-4155-b2f4-8571cea0c6bb.png)

Will be happy if you'll provide some suggestions to simplify the solution.

Reply:
@ryankeairns, what would you say about this issue? Do you have any suggestions? Thanks)

Reply:
@Kunzetsov apologies for the delay, endless meetings and other commitments. I'll take a look today.

Reply:
@ryankeairns, sure, no worries, take your time) Thanks for your efforts )

Reply:
Alright, I tried a couple of quick changes locally and recommend we do the following. 

First, remove this `filter.scss` file and, instead, do the following to achieve consistent padding:

1. Add the `compressed` prop to the `<EuiDescriptionList...`
2. Increase the padding on the parent `<EuiPanel...` from `s` to `paddingSize="m"`

This will result in a look like:

<img width="472" alt="Screen Shot 2021-11-11 at 8 43 23 AM" src="https://user-images.githubusercontent.com/446285/141317175-cdcc88d6-fc48-438d-b501-0411c0ff6e6b.png">

Reply:
Done)